exist/windows.md

```diff
@@ -0,0 +1,128 @@
+# Windows
+
+You have one disk with Windows installed and want to install macOS on the same disk.
+
+## Precautions
+
+- BACKUP YOUR DATA
+- If possible, disconnect or disable any other disk/drive in your system, as it may interfere with the install procedure
+- The drive isn't corrupted or have bad sectors
+- Stable power input
+- If you cannot add custom entries in your BIOS boot menu this method may not work for you, read all steps before proceeding
+
+## Situation this applies for
+
+- You have a disk with Windows installed
+- You have enough free space on the disk to support a macOS installation
+
+---
+
+Verify the partition style of the disk is GPT. If you do not know, see: [Checking your disk partition scheme](https://dortania.github.io/OpenCore-Multiboot/exist/data.html#checking-your-disk-partitioning-scheme) and if it is MBR, see: [Converting MBR to GPT](https://dortania.github.io/OpenCore-Multiboot/exist/data.html#converting-mbr-to-gpt).
+
+## Resize existing EFI partition
+
+If your EFI partition is bigger than the default of 100MB (at least 200MB to be safe) which is created by the Windows installation, then you can skip to the next section.
+
+![Windows before partitions](../images/ex-os/before_windows_disk_mgmt.png)
+
+Use a bootable partition manager to resize the EFI partition. To do this you must first move all the partitions in front of it so make sure to backup your data. 
+
+![Gparted original partitions](../images/ex-os/before_gparted.png)
+
+Moving a partition cannot be done while running Windows, so you will need a **bootable** partition manager (free alternative is GParted). This is a potentially destructive operation. 
+
+![Gparted warning](../images/ex-os/move_partition_warning_gparted.png)
+
+Backup your data and proceed at your own risk. If there is not enough free space to move existing partitions to the right, first shrink the existing Windows one. 
+
+![Gparted move Windows right](../images/ex-os/move_windows_right_gparted.png)
+
+Once there is enough free space after the EFI partition, extend it to at least 200MB. 
+
+![Gparted resize EFI](../images/ex-os/resize_efi_gparted.png)
+
```

Comment:
I would recommend against Gparted when dealing with ntfs, you can use Minitool or other known tools to do the same thing and they support ntfs from the OS, which is safer than using linux's ntfs driver.

Reply:
Removed mention of Gparted and update screenshots to Minitool instead